docs/en/core-services/Caching.md

```diff
@@ -155,9 +158,9 @@ For `ReadType.CACHE`, Alluxio will cache the block into the highest tier that ha
 So you will read the cache block with disk speed if the block is currently on a disk(SSD/HDD).
 
 > Note: In 2.3, Alluxio default ReadType was changed from `CACHE_PROMOTE` to `CACHE`.
-> This is because moving the block synchronously on reading will cause unnecessary delays.
-> In 2.3, we changed `CACHE` to utilize the [tiered storage management tasks](#tiered-storage-management)
-> to maintain block orders asynchronously. So the blocks will be promoted according to the
+> This is because moving the block to the top tier synchronously during a read will cause delays.
```

Comment:
I think cascading eviction is an implementation detail that the reader does not need to be so concerned about. It would probably require another section's worth of explanation that I don't have enough context to write for this update

I think delays is sufficient and doesn't bog the reader down with too many details.

Reply:
SGTM